Extension/src/Debugger/configurationProvider.ts

```diff
@@ -1162,16 +1162,16 @@ abstract class DefaultConfigurationProvider implements IConfigurationAssetProvid
 
 class WindowsConfigurationProvider extends DefaultConfigurationProvider {
     private executable: string = "a.exe";
-    private pipeProgram: string = "<" + localize("path.to.pipe.program", "full path to pipe program such as {0}", "plink.exe").replace(/\"/g, "\\\"") + ">";
+    private pipeProgram: string = "<" + localize("path.to.pipe.program", "full path to pipe program such as {0}", "plink.exe").replace(/"/g, '\\"') + ">";
```

Comment:
Most of the warnings are that the tool thinks we're trying to include an escape character in the search regex. We only care about fixing the double quotes since apparently the localizers add them in sometimes. It is not necesssary to escape the double quotes.